### PR TITLE
Update list with current Webrecorder related URLs

### PR DESCRIPTION
**Pull request by Shrinks99**
A few terms and links have changed!  These should all be the most current, Conifer is notably duped, could remove one of them?

### CODE REVIEW

**Comment by Shrinks99** [comment]
Makes sense to me!  Done.

Also added a few other cleanup fixes for Webrecorder tools.  Adds the `stable` badge to all, Browsertrix is exiting it's beta phase, ReplayWeb.page has been stable for a while, and Browsertrix Crawler is now in its V1!

File: README.md [hunk 1]
```diff
@@ -110,9 +109,9 @@ This list of tools and software is intended to briefly describe some of the most
 
 * [InterPlanetary Wayback (ipwb)](https://github.com/oduwsdl/ipwb) - Web Archive (WARC) indexing and replay using [IPFS](https://ipfs.io/).
 * [OpenWayback](https://github.com/iipc/openwayback/) - The open source project aimed to develop Wayback Machine, the key software used by web archives worldwide to play back archived websites in the user's browser. *(Stable)*
-* [PyWb](https://github.com/ikreymer/pywb) - A Python (2 and 3) implementation of web archival replay tools, sometimes also known as 'Wayback Machine'. *(Stable)*
+* [PYWB](https://github.com/webrecorder/pywb) - A Python (2 and 3) implementation of web archival replay tools, sometimes also known as 'Wayback Machine'. *(Stable)*
```
Comment (tw4l) [inline README.md:112]:
```suggestion
* [PYWB](https://github.com/webrecorder/pywb) - A Python 3 implementation of web archival replay tools, sometimes also known as 'Wayback Machine'. *(Stable)*
```

We finally dropped Python 2 support in the latest release put out today! haha

Comment (Shrinks99) [reply README.md:112]: This should also be updated in the readme ;)

**Comment by anjackson** [comment]
Thanks all! 👍

**Pull request merged**